test/e2e/tests/metrics/state-snapshots/errors-after-init-opt-in-background-state.json

```diff
@@ -267,11 +265,7 @@
     "manageInstitutionalWallets": "boolean"
   },
   "RemoteFeatureFlagController": {
-    "remoteFeatureFlags": {
```

Comment:
I don't understand all the changes made to these snapshots?

Reply:
This is what I got after running `yarn test:e2e:single test/e2e/tests/metrics/errors.spec.ts --browser chrome --update-snapshot` 😕 Will revert.